app/views/business_categories/show.html.haml

```diff
@@ -1,15 +1,32 @@
 %header.entry-header
-  %h1.entry-title= 'Företagstyp: ' "#{@business_category.name}"
+  %h1.entry-title
+    Hundföretag av typen
+    = @business_category.name
 .post-title-divider
   %span
 .entry-content
-  %p
-    %b Namn:
-    = @business_category.name
-  %p
-    %b Beskrivning:
-    = @business_category.description
-.center
-  = link_to 'Redigera', edit_business_category_path(@business_category)
-  \|
-  = link_to 'Alla kategorier', business_categories_path
+  - unless @business_category.description.nil?
+    %p
+      %b Beskrivning:
+      = @business_category.description
+  %table
+    %thead
+      %tr
+        %th Namn
+        %th Län
+        - if current_user.try(:admin)
+          %th Org nr
+          %th
+          %th
+    %tbody
+      - @companies.each do |company|
+        - if company_complete? company
+          - company.categories.each do |category|
+            - if category.name == @business_category.name
+              %tr.company
+                %td= link_to company.name, company
+                %td= company.region
+                - if current_user.try(:admin)
+                  %td= company.company_number
+                  %td= link_to 'Edit', edit_company_path(company)
+                  %td= link_to 'Delete', company, method: :delete, data: { :confirm => 'Are you sure?' }
```

Comment:
I have trouble following this logic.  

The Company model has this:
```
has_and_belongs_to_many :business_categories
```
but the BusinessCategory does not have the same association back to Company.

Here, we are looping through the categories associated with the employees of the company (via `Company#categories`).

So, it is unclear to me whether a Company should have a direct association with BusinessCategory, or no association and only be indirectly linked via employee's business categories, or both(?).

In the DB I am using, "Happy Mutts" has zero categories via employees and 2 categories directly:
```
2.3.1 :061 >   hm = Company.find_by_name('Happy Mutts')
 => #<Company id: 3, name: "Happy Mutts", company_number: "5562252998", phone_number: "", email: "kicki@gladajyckar.se", street: "Ålstensgatan 4", post_code: "123 45", city: "Bromma", region: "Stockholm", website: "http://www.gladajyckar.se", created_at: "2016-12-01 14:27:05", updated_at: "2016-12-01 14:27:05"> 
2.3.1 :062 > 
2.3.1 :063 >   hm.categories.count
 => 0 
2.3.1 :064 > hm.business_categories.count
 => 2 
2.3.1 :065 > 
```
Happy Mutts, of course, shows zero categories in this `show` view.

Do we need/want two types of business category associations for a company?  Please advise.